README.md

```diff
@@ -13,6 +13,10 @@ this module may vary (username/password, SAML, OAuth, etc.), and it is possible
 for more than one Authentication module to exist in a running system. The
 default implementation uses a simple username and password for authentication.
 
+# Environment variables
+
+* LOGIN_COOKIE_SAMESITE - Configures the SameSite attribute of the login token cookies. Must be `Strict`, `Lax` or `None`. Defaults to `Lax` if not set. `None` must be used if front-end and backend (Okapi) are served from different host names to allow non-GET requests like POST, PUT and DELETE. If served from the same host name `Lax` allows deep links from other sites, for example from a wiki or webmail to an inventory instance record, whereas `Strict` doesn't allow them.
```

Comment:
This statement:

> `None` must be used if front-end and backend (Okapi) are served from different host names to allow non-GET requests like POST, PUT and DELETE.

I'm not sure if we know that yet until we test. Do you want perhaps to remove it for now?